src/js/mchad.js

```diff
@@ -60,9 +58,20 @@ export async function getArchiveFromRoom(room) {
       link: room.Link
     })
   }).then(toJson).catch(() => []);
-  const { Stime: firstTime } = [...script, { Stime: 0 }, { Stime: 0 }][1];
+  
+  var { Stime: firstTime } = [...script, { Stime: 0 }][0];
```

Comment:
Please use ``const`` or ``let`` instead of ``var``.

Reply:
done